open_ce/utils.py

```diff
@@ -53,6 +53,7 @@
 CONTAINER_TOOLS = ["podman", "docker"]
 DEFAULT_CONTAINER_TOOL = next(filter(lambda tool: os.system("which {} &> /dev/null".format(tool))
                                       == 0, CONTAINER_TOOLS), None)
+DEFAULT_PKG_FORMAT = "2"  # use conda_v2 output format
```

Comment:
We need to think about what the default should be. Maybe start with the tar.bz2 format for now, then we can play around with the new option for a bit to see if there are any issues with switching.

Reply:
Sure, any value other than 2 as of now means the .tar.bz2 format. So shall we set it to 1. 
https://github.com/conda/conda-build/blob/master/conda_build/config.py#L219

Reply:
I have used "tarball" and "conda" as two options. @bnemanich, please let me know if this looks fine. 

Once we finalize this, I will update the documentation.